### ISSUE TITLE
Graph Level Output

### ISSUE THREAD

**Issue by Faiq-Ghawash** [ASzot/ggnn#1]
I want to ask that, is it possible to get the graph level output using this code. As you indicated in your code  
  ```
Equation 7 of section 3.3
        output = self.out(torch.cat((hidden_state, annotation), 2))
        output = output.sum(2)
```
This must evaluate the graph level representation of the graph. But i think this outputs the node scores using  the following equation described in the paper
`o = g(h; x)`
and then passing these scores in cross entropy loss where first softmax is applied and then the loss is evaluated. I am confused, can you please guide a bit on this aspect.
Moreover, if it provides a graph level representation vector, then for graph classification task i can take this vector as input to another neural network that will output the class the graph belongs to and that thing is not yet available with the current code?
